test/cts/test/V1_0/resize_bilinear.js

```diff
@@ -2,7 +2,7 @@ describe('CTS', function() {
   const assert = chai.assert;
   const nn = navigator.ml.getNeuralNetworkContext();
 
-  it('check result for Resize bilinear example', async function() {
+  it('check result for Resize bilinear with inputs(without align_corners) example', async function() {
```

Comment:
Tiny but please add a space between words "inputs" and "(". Same for others.

Reply:
@ibelem The purpose of 'inputs(without align_corners)' text is clear to qualify 'inputs', so l skip that space. :)